src/main/java/com/singlestore/r2dbc/codec/DataType.java

```diff
@@ -32,14 +32,25 @@ public enum DataType {
   BLOB(252),
   VARSTRING(253),
   STRING(254),
-  GEOMETRY(255);
+  GEOMETRY(255),
```

Comment:
unrelated to PR: do you plan to remove GEOMETRY, or to modify it to correspond to our Geography(point), or just leave it as is?